### PR TITLE
ROX-13742: Add notes to node inventory

### PR DESCRIPTION
**Pull request by daynewlee**
## Description

Add "language scanning not supported" notes in the communication from compliance to sensor

### CODE REVIEW

**Comment by daynewlee** [comment]
Manual test result:

```
common/compliance: 2023/01/04 16:51:35.253690 service_impl.go:105: Info: Received connection from "gke-yl-01-04-disgusting--default-pool-83c330ed-ctnp"
common/compliance: 2023/01/04 16:51:36.235906 service_impl.go:105: Info: Received connection from "gke-yl-01-04-disgusting--default-pool-83c330ed-ctnp"
common/compliance: 2023/01/04 16:51:36.238048 node_inventory_handler_impl.go:94: Info: Current node inventory [gke-yl-01-04-disgusting--default-pool-83c330ed-ctnp] note is: LANGUAGE_CVES_UNAVAILABLE
common/signal: 2023/01/04 16:51:36.752743 signal_service.go:107: Error: error dequeueing signalStreamMsg event: rpc error: code = Canceled desc = context canceled
pkg/kocache: 2023/01/04 16:51:39.915513 source.go:66: Info: loading probe 2.3.0-rc1/collector-ebpf-5.4.0-1086-gke.o.gz from upstream /kernel-objects
common/signal: 2023/01/04 16:51:39.961393 signal_service.go:103: Info: starting receiveMessages
common/compliance: 2023/01/04 16:51:41.871954 service_impl.go:105: Info: Received connection from "gke-yl-01-04-disgusting--default-pool-83c330ed-vv0n"
common/compliance: 2023/01/04 16:51:42.812863 service_impl.go:105: Info: Received connection from "gke-yl-01-04-disgusting--default-pool-83c330ed-vv0n"
common/compliance: 2023/01/04 16:51:42.814412 node_inventory_handler_impl.go:94: Info: Current node inventory [gke-yl-01-04-disgusting--default-pool-83c330ed-vv0n] note is: LANGUAGE_CVES_UNAVAILABLE

```

File: compliance/collection/nodeinventorizer/nodeinventory.go [hunk 1]
```diff
@@ -37,6 +37,11 @@ func (n *NodeInventoryCollector) Scan(nodeName string) (*storage.NodeInventory,
 		ScanTime:   timestamp.TimestampNow(),
 		Components: protoComponents,
 	}
+	// uncertifiedRHEL is false since scanning is only supported on RHCOS for now,
```
Comment (Maddosaurus) [inline compliance/collection/nodeinventorizer/nodeinventory.go:40]: Nit: As we add this very note to every scan for now, we could also move it into the message init, as opposed to adding it afterwards, right?
```go
	m := &storage.NodeInventory{
		NodeName:   nodeName,
		ScanTime:   timestamp.TimestampNow(),
		Components: protoComponents,
                Notes:      []scannerV1.Note{scannerV1.Note_LANGUAGE_CVES_UNAVAILABLE},
	}
```

**Review by Maddosaurus — APPROVED** [review]
That looks good to me!
I have a nit with regards to readability, but that's non-blocking

File: compliance/collection/nodeinventorizer/nodeinventory.go (hunk 1 above)
Comment (vikin91) [reply compliance/collection/nodeinventorizer/nodeinventory.go:40]: I second what Matthias wrote - I would also see it inside the `storage.NodeInventory` initialization block.

**Pull request merged**